dashboard/src/features/shared/components/SearchBar/SearchBar.jsx

```diff
@@ -167,6 +167,9 @@ class SearchBar extends React.Component {
             </span>
           </span>}
         </span>
+          <a href="/docs/core/build-applications/queries" target="_blank" className={styles.viewdocs}>
+            Queries
```

Comment:
I don't think this label fits very well - might need a little @jeffomatic wordsmithing.